providers/PaymasterProvider.tsx

```
@@ -0,0 +1,46 @@
+  const { address } = useConnectWallet()
+
+  const { data: availableCapabilities } = useCapabilities({
+    account: address,
+  })
+
+  const getCapabilities = (chainId: number) => {
+    if (!availableCapabilities || !chainId) return {}
+    const capabilitiesForChain = availableCapabilities[chainId]
+    if (
+      capabilitiesForChain['paymasterService'] &&
+      capabilitiesForChain['paymasterService'].supported
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)